blocks/providers/providers.css

```diff
@@ -0,0 +1,81 @@
+.providers{
+    padding: 0 10px;
```

Comment:
Padding looks great on mobile, just make sure to update for larger screens.

Reply:
Spacing on the bottom of the block is inaccurate on the larger sizes. Right now it's determined by the margins of the card text but it doesn't update as the screen widens and remains at 40px instead of increasing to 100px. Preference here, but I think a block level padding would provide more consistent spacing instead of margins on elements within the block. Keeps consistency of block level spacing if things were to move around later.